Assets/Scripts/Buildings/Upgrades/HealthUpgrade.cs

```diff
@@ -0,0 +1,26 @@
+using UnityEngine;
+
+namespace Buildings.Upgrades
+{
+    [RequireComponent(typeof(Building))]
```

Comment:
Kann das nicht in die base-class?

Reply:
Mittlerweile ja 😅 Hatte ich mal anders geplant, aber jetzt ist's sinnvoll